### DISCUSSION TITLE
Could I use Chromium on it?

### DISCUSSION THREAD

**Discussion by DaryeDev** [ThomasPrior/LenovoSmartClock2#9] [Q&A] [answered]
Looking to have a cool mini dashboard on my desk, and a second hand Lenovo Smart Clock might look cool. 

How does it run a modern version of Chromium?

**Comment by ThomasPrior** [comment, marked as answer]
Don't know, chromium is not something I've attempted to run.

Chrome installs and works just fine. Follow the steps to install an updated webview: https://github.com/ThomasPrior/LenovoSmartClock2#install-updated-webview-component

**Comment by DaryeDev** [comment]
Chrome works for me, thanks for the fast response!
